astro/src/layouts/EscapeRoomLayout.astro

```diff
@@ -0,0 +1,456 @@
+            </p>
+          </div>
+          <div class="col-md-4">
+            <h3 class="h5">Quick Links</h3>
+            <ul class="list-unstyled">
+              <li><a href="/services">Services</a></li>
+              <li><a href="/about">About Us</a></li>
+              <li><a href="/volunteer">Join us</a></li>
+            </ul>
+          </div>
+          <div class="col-md-4">
+            <h3 class="h5">Contact Us</h3>
+            <p>
+              Have questions about our services?<br />
+              <a href="mailto:info@accessiblecommunity.org">info@accessiblecommunity.org</a>
+            </p>
+          </div>
+        </div>
+        <hr />
+        <div class="text-center">
+          <p class="mb-0">
+            &copy; {new Date().getFullYear()} <Branding>Accessible Community</Branding>. All rights reserved.
+          </p>
+        </div>
+      </div>
+    </footer>
+
+    <script src="../scripts/headings.js"></script>
+    <script src="../scripts/body-no-scroll.js"></script>
+    <script>
+      // Simple toggle for navbar
+      document.addEventListener('DOMContentLoaded', function() {
+        const toggler = document.getElementById('navToggler');
+        const menu = document.getElementById('navbarNav');
+        
+        if (toggler && menu) {
+          // Toggle menu on button click
+          toggler.addEventListener('click', function(e) {
+            e.stopPropagation();
+            const isExpanded = toggler.getAttribute('aria-expanded') === 'true';
+            toggler.setAttribute('aria-expanded', !isExpanded);
+            menu.classList.toggle('show');
+          });
+          
+          // Close menu when clicking elsewhere
+          document.addEventListener('click', function(e) {
+            if (!menu.contains(e.target) && !toggler.contains(e.target) && menu.classList.contains('show')) {
+              menu.classList.remove('show');
+              toggler.setAttribute('aria-expanded', 'false');
+            }
+          });
+          
+          // Close menu when clicking a link
+          const links = menu.querySelectorAll('.nav-link');
+          links.forEach(link => {
+            link.addEventListener('click', function() {
+              menu.classList.remove('show');
+              toggler.setAttribute('aria-expanded', 'false');
+            });
+          });
+        }
+      });
+    </script>
+    {
+      import.meta.env.PROD && (
+        <script
+          is:inline
+          defer
+          data-domain="accessiblecommunity.org"
+          src="https://plausible.io/js/script.js"
+        />
+      )
+    }
+  </body>
+</html>
+
+<style>
+  /* Custom Header Styles */
+  .escape-room-header {
+    background: linear-gradient(135deg, var(--bs-primary) 0%, #2c3e50 100%);
+    padding: 1rem 0;
+    box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.1);
+    position: relative;
+    z-index: 1000;
```

Comment:
You shouldn't need a z-index this high. I believe z-index of 10 or 20 should work.